client/src/app/site/pages/organization/pages/committees/pages/committee-detail/modules/committee-detail-edit/components/committee-detail-edit/committee-detail-edit.component.ts

```diff
@@ -120,7 +120,7 @@ export class CommitteeDetailEditComponent extends BaseComponent implements OnIni
             const identifiable = (await this.committeeRepo.create(value))[0];
             id = identifiable.id;
         } else {
-            await this.committeeRepo.update(value, this.editCommittee);
+            (await this.committeeRepo.update(value, this.editCommittee).resolve()) as unknown as Promise<void>;
```

Comment:
really ugly cast... Shouldn't this be automatically recognized as `Promise<void>`? If not, is there no other way to solve this? Why do you even need to cast this?

Reply:
Sadly - for some reason - it isn't. The return type of .resolve() is Promise<void | void[]>, that causes errors.
The reason I did this is precisely because I couldn't think of another way to resolve this.

Reply:
Do you have an idea how I could do this?

Reply:
Just remove the casts completely since you don't need the type anywhere:
```suggestion
            await this.committeeRepo.update(value, this.editCommittee).resolve();
```
At least my editor does not complain then ;) did not try to compile